src/BTVNanoCommissioning/workflows/ctag_Wctt_valid_sf.py

```diff
@@ -246,11 +252,11 @@ def process_shift(self, events, shift_name):
         )
 
         dilep_mass = iso_lep + soft_muon[:, 0]
-        if isMu:
+        if isMu and "noMuVeto" not in self.selMod:
             req_dilepmass = (dilep_mass.mass > 12.0) & (
                 (dilep_mass.mass < 80) | (dilep_mass.mass > 100)
             )
-        elif isEle:
+        else:
```

Comment:
@adeiorio could you please check whether this modification consider the WP workflow?